x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/date.tsx

```diff
@@ -94,9 +94,15 @@ export const DateProcessor: FunctionComponent = () => {
         )}
       />
 
-      <UseField config={fieldsConfig.formats} component={ComboBoxField} path="fields.formats" />
+      <UseField
+        data-test-subj="formatsValueField"
+        config={fieldsConfig.formats}
+        component={ComboBoxField}
+        path="fields.formats"
+      />
 
       <TargetField
+        data-test-subj="targetFieldsField"
```

Comment:
nit: it looks like this `data-test-subj` is not used, as `TargetField` has it hard coded to `targetField`.

Reply:
Will remove.